src/components/kaltura/KalturaEventList.vue

```diff
@@ -15,26 +15,22 @@
     hide-default-footer
     :items="events"
     :items-per-page="-1"
+    :loading="isLoading"
     no-data-text="No Kaltura events."
   >
     <template #item.summary="{item}">
-      <a
-        :id="`kaltura-media-space-${item.id}`"
-        :href="`${contextStore.config.kalturaMediaSpaceUrl}/recscheduling/index/edit-event/eventid/${item.id}`"
-        target="_blank"
-      >
-        {{ item.summary }} <span class="sr-only">(opens in new tab)</span>
-        <v-icon class="ml-1" :icon="mdiOpenInNew" size="14" />
-      </a>
+      <ExternalLink :href="`${contextStore.config.kalturaMediaSpaceUrl}/recscheduling/index/edit-event/eventid/${item.id}`" :link-id="`kaltura-media-space-${item.id}`">
```

Comment:
@lyttam - Is "recscheduling" (in URL above) a typo?

Reply:
@johncrossman no - this tripped me up too. It must stand for "recording scheduling".